_data/ball/2022.yaml

```diff
@@ -32,14 +30,7 @@ faq:
   - question: What will be there?
     answer: Balls usually have a 3 course meal, a bar, and a dancefloor. We'll have all that, a live DJ, and a photobooth.
   - question: When can I buy tickets?
-    answer: Tickets are live now! Use the link at the top of the page.
-  - question: The ticket link isn't working :(
-    answer: Make sure you're logged in to the Guild website, otherwise it'll say that the content is unavailable. <br><br>You also need to be a member to buy a ticket. If you're not one already you can go to <a href="https://cssbham.com/join">cssbham.com/join</a> to become a member
-  - question: How much will tickets cost?
-    answer: Tickets will cost £42.50. The Guild will also add on a booking fee - we can't do anything about that.
-  - question: Can I bring a guest?
-    answer: Yes, though they'll need a guest ticket to attend - these will be available for purchase on Thursday 14th April.
-  - question: Will I need to show my ticket when I arrive?
+    answer: Tickets are now closed for the 2022 Ball.
```

Comment:
I think it would be better to not delete any questions, so only changing the answers. 

Questions staying would be useful in the future to look back, and some are still relevant Qs

Reply:
e.g.

```
- question: Can I bring a guest?
    answer: Of course! Guests are more than welcome to attend.
```